py/ai/h2o/sparkling/ml/params/H2OTargetEncoderParams.py

```diff
@@ -56,7 +56,7 @@ def getInputCols(self):
         return self.getOrDefault(self.inputCols)
 
     def getOutputCols(self):
-        return map(lambda c: c + "_te", self.getInputCols())
+        return list(map(lambda c: c + "_te", self.getInputCols()))
```

Comment:
on Python3 this returns object, we need to pass it into list method, which returns list on python2 and python3